packages/rpc/src/binance/models/BinanceNodeInfo.ts

```diff
@@ -0,0 +1,45 @@
+import 'reflect-metadata';
+import { Type, Transform } from 'class-transformer';
+import BigNumber from 'bignumber.js';
+ 
+class NodeInfo {
+    network: String
+    listen_addr: String
```

Comment:
Please use camelcase instead of snake case. To transform the data into camelcase we use `@Expose` annotation. In this case we should have:
```
@Expose({ name: 'listen_addr' })
listenAddr: string